asv_bench/benchmarks/benchmarks.py

```diff
@@ -57,8 +55,8 @@ def setup(self, shape, ngroups=5, groupby_ncols=1):
 class TimeGroupByMultiColumn(BaseTimeGroupBy):
     param_names = ["shape", "ngroups", "groupby_ncols"]
     params = [
-        UNARY_OP_DATA_SIZE[ASV_DATASET_SIZE],
-        GROUPBY_NGROUPS[ASV_DATASET_SIZE],
+        get_benchmark_shapes("TimeGroupByMultiColumn"),
```

Comment:
@anmyachev , you did huge changes :) I didn't look them yet though but can't we use a simpler approach?

```suggestion
        get_benchmark_shapes("TimeGroupByMultiColumn") or UNARY_OP_DATA_SIZE[ASV_DATASET_SIZE],
```
`get_benchmark_shapes` would return shapes from the file or None. Or do you think it is worth considering your changes?

Reply:
In general, our solutions are similar, but in my case, the data sizing mechanism becomes centralized, which in turn makes it easier to make future changes and support.